### ISSUE TITLE
Integrated with Direct Message Instagram 

### ISSUE THREAD

**Issue by reyeff** [junhoyeo/threads-api#100]
Hello it is possible this repo can read direct message from Instagram Account. So the message can be post it on Threads, maybe with some filter like if the Account is followed by Instagram/Threads and filter message. 

Im happy if it can. Thank you✌️

**Comment by cedriking** [comment]
This is probably something to be in `Built with`, I don't think this should part of threads-api as it's not related to threads.

**Comment by SethuSenthil** [comment]
I think it's kinda out of scope for this library as this primarily focuses on threads. There are other libraries that focus on the Instagram API like this one so I don't think there is any purpose in reinventing the wheel:
https://github.com/dilame/instagram-private-api

You can write a simple node js or deno program using both of these libraries together to achieve what you are looking for. 
But ultimately it depends on what the other contributors think!

**Comment by cedriking** [comment]
I agree that it feels out of the scope of the API. Sounds more like a "Built with" option.
